stdlib/messaging/activemq-artemis/src/main/ballerina/artemis/listener.bal

```diff
@@ -41,22 +47,33 @@ public type Listener object {
         return self.createConsumer(serviceType);
     }
 
-    function start() returns error? = external;
-    function createConsumer(service serviceType) returns error? = external;
-    function stop() returns error? = external;
+    # The Artemis consumer is represented by the Service object. 
+    # But if there is a need to make blocking recieve calls, probably within a transaction block 
+    # this method can be used to create and get a `Consumer` object.
+    # 
+    # + config - the configuration for the queue to consume from
```

Comment:
```suggestion
    # + config - the configuration consumed by the queue
```

Reply:
It is the config for the queue not config consumed by queue :-1: